### PR TITLE
feat(nc-gui): select multiple rows and copy

### PR DESCRIPTION
**Pull request by elvus**
## Change Summary
Add new file:
- packages/nc-gui/composables/useMultiSelect/index.ts

Update: 
- packages/nc-gui/components/smartsheet/Grid.vue
## Change type

- [x] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Additional information / screenshots (optional)

https://user-images.githubusercontent.com/24416449/192377362-c86a6539-6297-44b8-b0c6-364cf0a941de.mp4

### CODE REVIEW

**Comment by elvus** [comment]
Cc @bcakmakoglu @o1lab

**Review by bcakmakoglu — APPROVED** [review]

**Comment by bcakmakoglu** [comment]
@pranavxc Can you take a quick peek here as well and confirm that this is good to merge? Thanks =)

File: packages/nc-gui/composables/useMultiSelect/index.ts [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+          let cptext = '' // variable for save the text to be copy
+
+          if (!isNaN(rangeRows.minRow) && !isNaN(rangeRows.maxRow) && !isNaN(rangeRows.minCol) && !isNaN(rangeRows.maxCol)) {
+            const cprows = unref(data).slice(rangeRows.minRow, rangeRows.maxRow + 1) // slice the selected rows for copy
+
+            const cpcols = unref(fields).slice(rangeRows.minCol, rangeRows.maxCol + 1) // slice the selected cols for copy
+
+            cprows.forEach((row) => {
+              cpcols.forEach((col) => {
+                cptext = `${cptext} ${row.row[col.title]} \t`
```
Comment (pranavxc) [inline packages/nc-gui/composables/useMultiSelect/index.ts:216]: - LTAR and Lookup column value will copy as  `[object Object]` 
- Attachment will be JSON string or `[object Object]`

**Review by pranavxc — APPROVED** [review]

File: packages/nc-gui/composables/useMultiSelect/index.ts (hunk 1 above)
Comment (bcakmakoglu) [reply packages/nc-gui/composables/useMultiSelect/index.ts:216]: Should these be filtered out?

Comment (pranavxc) [reply packages/nc-gui/composables/useMultiSelect/index.ts:216]: Either we can filter or we need to extract the primary value and concatenate. Or even we can convert to JSON string to begin with and later we can do the primary value extracton.

Comment (bcakmakoglu) [reply packages/nc-gui/composables/useMultiSelect/index.ts:216]: @elvus Let me fix the copy issue for the mentioned cells and then we can get this merged :)

**Pull request merged**